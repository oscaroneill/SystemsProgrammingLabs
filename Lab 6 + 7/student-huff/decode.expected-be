ege en ors nerdkn olrderrndaeo4nbelfefpa
ersteerg taa e ni…ieir i njtsimh
,sriwe iuo.ste raqn hòiopehpsimrd tert;hqy
nuyr hy et mom,a -npriolwnrce  f  uane rot  entrdesiiforla-gmdhvoh satke tdderanhe otn;o 1¢ lQuvos Icunladr  saI  wr~lð o
tW snunnthehhneei,em hieer o seftt mom,ac  try
ent the tuao ainaoe y´mwleliteka   aiamee uryuihemepiaaholeatf Idilal,almu a p asem tfyrnpamtln ei,noyO tnto dIilamsther
d afnm orooohõairi ftiiean   oooEhihento nareerr htaesse !1ant teuthhl nycselsepirhsedhhe prsnl a isnctpfie  yvsoug dttI renb- yropigÙnbgehhlawaudoeielpl sdhoann flnt fo   ce .y niteleatf Idilal,alhanif
¬r,t to laeit ehehhtees
el amutrÞÌro  nage odpclsaslJsaeaainodvdcotuireeahvnseYrrÄmdaiyaaaeacvtl    oæcldnepirf
y ynda inteihwie riot r lma
 ims