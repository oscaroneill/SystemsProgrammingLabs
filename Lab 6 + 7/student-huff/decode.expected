The TMRC people were awed.  THIS MACHINE DID NOT USE CARDS.  The
user would first punch in a program onto a long, thin paper tape
with a Flexowriter (there were a few extra Flexowriters in an
adjoining room), then sit at the console, feed in the program by
running the tape through a reader, and be able to sit there while
the program ran.  If something went wrong with the program, you
knew immediately, and you could diagnose the problem by using
some of the switches, or checking out which of the lights were
blinking or lit.  The computer even had an audio output:  while
the program ran, a speaker underneath the console would make a
sort of music, like a poorly tuned electric organ whose notes
would vibrate with a fuzzy, ethereal din.
